src/components/Span/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import {
+  SpanItem
+} from './styles';
+
+export default function Span({ errorType }) {
+  const type = errorType === '1' ? 'Campos Ausentes' : 'Atributos Ausentes';
```

Comment:
Poderíamos colocar numa constante esse valor '1', pra dar um nome pra ele, tipo

```suggestion
  const MISSING_FIELDS = '1';
  const type = errorType === MISSING_FIELD ? 'Campos Ausentes' : 'Atributos Ausentes';
```

O que tu acha?

Reply:
Ou um método que retorne a string com base no código, `function getErrorMessage(code: string): string`, mas talvez seja demais pra um ternário :laughing:

Reply:
boaa, faltou só isso